index.html

```diff
@@ -70,6 +70,163 @@ <h1>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="about-me">
+      <h2>About Me</h2>
+      <img src="./images/Mask Group.png" class="background-top-right" alt="">
+      <img src="./images/Frame.svg" class="background-bottom-left" alt="">
```

Comment:
- [ ] You did a great job, :+1: Your UI looks great like the given design. But there are some project requirements and best practices you need to improve on before moving to the next milestone. :100:

- - These images here according to the design are background images that are meant for design. Following the best practices, decorations are always backgrounded images, not `img tags`. I'll suggest you remove these image tags here and implement the background image in your CSS. :smile: :wave:

Reply:
@V-Blaze - Solved and this is the layout after resolving 
![Screenshot from 2023-06-02 14-25-13](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/8fb39e3b-5d23-47dc-bd5f-93d7fbf6c027)